source/MaterialXTest/GenGlsl.cpp

```diff
@@ -92,5 +90,56 @@ TEST_CASE("GLSL Unique Names", "[genglsl]")
     mx::ShaderGeneratorPtr shaderGenerator = mx::GlslShaderGenerator::create();
     shaderGenerator->registerSourceCodeSearchPath(searchPath);
 
-    testUniqueNames(shaderGenerator, mx::Shader::PIXEL_STAGE);
+    GenShaderUtil::testUniqueNames(shaderGenerator, mx::Shader::PIXEL_STAGE);
+}
+
+class GLSLGenCodeGenerationTester : public GenShaderUtil::ShaderGeneratorTester
```

Comment:
Each derived tester looks pretty simliar. The GLSL/OGSFX one is a bit different since there are some nodedefs which we don't support yet so indicate to skip testing them.